Doc/library/tokenize.rst

```diff
@@ -17,7 +17,7 @@ as well, making it useful for implementing "pretty-printers," including
 colorizers for on-screen displays.
 
 To simplify token stream handling, all :ref:`operators` and :ref:`delimiters`
```

Comment:
Could you also address @terryjreedy's comment on the tracker and change

```rst
:ref:`operators` and :ref:`delimiters`
```

to

```rst
:ref:`operator <operators>` and :ref:`delimiter <delimiters>`
```

Reply:
Fixed